gatsby/static/css/how-it-works.css

```diff
@@ -13,8 +13,9 @@
 .legend {
     position: absolute;
     padding-top: 20px;
-    opacity: 0;
     max-width: 570px;
+    opacity: 0;
+    pointer-events: none;
```

Comment:
Looks good to me 👍

https://user-images.githubusercontent.com/558581/167490575-a9279ffb-6bec-412b-a494-414c094344df.mp4